rbd/features.go

```diff
@@ -74,21 +74,32 @@ const (
 	FeatureNameDataPool = C.RBD_FEATURE_NAME_DATA_POOL
 
 	// old names for backwards compatibility (unused?)
-	RbdFeatureLayering      = FeatureLayering
-	RbdFeatureStripingV2    = FeatureStripingV2
-	RbdFeatureExclusiveLock = FeatureExclusiveLock
-	RbdFeatureObjectMap     = FeatureObjectMap
-	RbdFeatureFastDiff      = FeatureFastDiff
-	RbdFeatureDeepFlatten   = FeatureDeepFlatten
-	RbdFeatureJournaling    = FeatureJournaling
-	RbdFeatureDataPool      = FeatureDataPool
 
+	// RbdFeatureLayering deprecated alias for FeatureLayering
```

Comment:
Comments for deprecation should follow a specific format as that is detected by `go vet` (or was it an other tool?). See `rbd/rbd.go:Open()`.

Reply:
I don't know if that makes sense for one-liners like these. If the tool can only detect "Deprecated:" at the start of a line that breaks the stronger standard of starting with the items name. I'd really prefer not to have to make these multi-line.